API.md

```diff
@@ -0,0 +1,84 @@
+OpenMoji API
+========
+
+Access OpenMoji emoji images, their metadata, and colors from your Node.js application.
+
+## Install
+
+Install with [npm](https://www.npmjs.com/package/openmoji):
+
+    $ npm install openmoji
+
+## Usage
+
+    > const openmoji = require('openmoji')
+    > const moji = openmoji.openmojis[0]
+    > moji.emoji
```

Comment:
I would rather go with `emoji`, `openmoji`, `om`, `o` and not with `moji`.

Reply:
I aimed to provide distinction between the emoji the character and the emoji the datum. What is the singular form of "apples"? Apple. What is the singular form of "openmojis"? Openmoji. But we cannot say "openmoji" because the package has already reserved that name. The package could be assigned into `om` or something else like `const o = require('openmoji')` but I think it is a good habit to allow developers write `const openmoji = require('openmoji')`. It reduces the number of names to remember.

I think `om` as an emoji data unit sounds fun. I would choose it.

    const openmoji = require('openmoji')
    const om = openmoji.openmojis[0]
    om.emoji

Small things but these will travel far once established. Thoughts?

Reply:
Very happy with om. Many thanks for sharing your rationale behind it! I feel this entire OpenMoji things is all about details :)